unicorecmsebola/static/css/style.css

```diff
@@ -76,6 +76,10 @@
 	#menu { background: #d15c4d; color: #f8f7e7; text-align: center; }
 		#menu a { padding: 8px 10px; display: block; border-bottom: 1px solid #fff; text-align: center; color: #f8f7e7; }
 
+/* BASE */
```

Comment:
Ahem. Yeah, i think it was the MNM CSS that had that as the descriptor, so I was trying to be consistent. On reflection that's consistently bad.